packages/lib-sourcify/src/Verification/VerificationTypes.ts

```diff
@@ -95,7 +97,10 @@ export interface VerificationExport {
     sources: StringMap;
     compilerOutput: {
       // The export should not include the AST object to reduce the size
-      sources?: Record<string, { id: number }>;
+      sources?: Record<
+        string,
+        Pick<SolidityOutputSource, 'id'> | Pick<VyperOutputSource, 'id'>
```

Comment:
Does it make sense from a type point of view to say number or number? It seems like it's always number.

Reply:
Do you mean to just use `Record<string, {id: number}>`?